SteinsKit/Observations/AnyObservable.swift

```diff
@@ -90,4 +108,12 @@ extension AnyObservable: Observable {
         return box.beObserved(by: observer, method, onChanged: handler)
     }
     
+    public func beObserved<Observer>(by observer: Observer, disposer: AnyObject?, onChanged handler: @escaping (Observer, Value) -> Void) where Observer : AnyObject {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/el-hoshino/SteinsKit/blob/6953eff541e91809c29fd6259d7811df88c4de25/SteinsKit/Observations/AnyObservable.swift#L111"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>AnyObservable.swift:111</code></td>
    </tr>
  </tbody>
</table>